cmd/upgrade/accountroles/cmd.go

```diff
@@ -251,6 +295,144 @@ func LogError(key string, ocmClient *ocm.Client, defaultPolicyVersion string, er
 	}
 }
 
+func upgradeAccountRolePoliciesFromCluster(
+	reporter *rprtr.Object,
+	awsClient aws.Client,
+	cluster *v1.Cluster,
+	accountID string,
+	policies map[string]string,
+	policyVersion string,
+	policyPath string,
+	isVersionChosen bool,
+) error {
+	for file, role := range aws.AccountRoles {
```

Comment:
This we cannot do it for the older formats since they dont follow our standards

Reply:
We can probably go over this, but I'm using those as a way to setup a map to the roles defined within the cluster

Reply:
Yes, please. We need not use the prefix at all. The flow should be simple for cluster upgrades. Just fetch the roles from CS DB and update. So we need to take out all the prefix logic .